src/components/ErrorBoundary.jsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import logError from '../utils/logError';
+
+// ? We're not able to use SFC's and hooks with componentDidCatch: https://reactjs.org/docs/hooks-faq.html#do-hooks-cover-all-use-cases-for-classes
+export default class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { error: props.forceError };
+  }
+
+  componentDidCatch(error, errorInfo) {
+    this.setState({ error });
+    logError({ error, errorInfo });
+  }
+
+  // ? This is purely here for testing purposes
+  componentDidMount() {
+    this.props.forceError &&
+      logError({
+        error: new Error('Test errors'),
+        errorInfo: { componentStack: 'ErrorBoundary' },
+      });
+  }
```

Comment:
Not sure how I feel about this, but was getting tired of looking for a reliable way to get jest to chill out and let me trigger errors... So I threw in a prop to simulate the behavior.